tester/src/test/java/com/silanis/esl/sdk/examples/ExpiryTimeConfigurationExampleTest.java

```diff
@@ -0,0 +1,22 @@
+package com.silanis.esl.sdk.examples;
+
+import com.silanis.esl.sdk.ExpiryTimeConfiguration;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class ExpiryTimeConfigurationExampleTest {
+
+    @Test
+    public void verifyResult() {
+        ExpiryTimeConfigurationExample example = new ExpiryTimeConfigurationExample();
+        example.run();
+
+        ExpiryTimeConfiguration expiryTimeConfiguration = example.getExpiryTimeConfigurationAfterUpdate();
```

Comment:
This example for the integration user. It's kind of a document. If possible, avoid encapsulation and set the variable to be public so that it can be accessed directly in the example test if necessary.